requirements/usage/usage-scenarios.tex

```diff
@@ -0,0 +1,26 @@
+\subsection{Usage scenarios}
+\begin{enumerate}
+	\item High priority task.
+		\begin{itemize}
+			\item Bob has to do some urgent simulations.
+			\item Bob inputs the task and data, and sets the priority option to high.
+			\item System checks if the task can be directly executed somewhere.
+			If that is the case, the task is executed and the scenario ends.
+			\item System tries to find a running task with lower priority than Bob's task, so that Bob's task can be executed at its place.
+			If that is the case, the task with lower priority is stopped and replaced with Bob's.
+			\item If all running tasks are with equal or greater priority, the User is notified that he has to wait.
+		\end{itemize}
+	\item User checks the state of the task.
+		\begin{itemize}
+			\item Bob starts a new simulation, that he has never run before.
+			Bob has some small concerns that there will be errors in the simulation.
+			\item System has available resources, so it starts the simulation.
+			\item After 15 minutes, Bob checks how the simulation is going.
+			\item System gives Bob the possibility to inspect the running simulation.

```

Comment:
`possibility` doesn't sound well, and Cambridge dictionary seems to agree: https://dictionary.cambridge.org/grammar/british-grammar/opportunity-or-possibility

Anyway, I'd recommend using a simple verb, like `enables` or `lets`